c-pallets/segment-book/src/lib.rs

```diff
@@ -645,6 +658,7 @@ pub mod pallet {
 		}
 
 		//Clean up the corresponding challenges in the miner's challenge pool
+		// You are trusting that the whatever is using the method has checked Vec limits already
 		fn clear_challenge_info(
 			miner_acc: AccountOf<T>,
 			prove_list: Vec<ProveInfo<T>>,
```

Comment:
To be on the safe side, I chose `BoundedVec`:
https://github.com/CESSProject/cess/pull/88/files#diff-e315e38811b02c01aead9143bcfe8a7d0db178f24a154bad1431a08648393296R677